### PR TITLE
Add single unit test and yml file for travis ci

### PR DESCRIPTION
**Pull request by keiththompson**
Added a single unit test & yml file to run builds on Travis. 
Addresses this issue: https://github.com/ragnraok/RxCamera/issues/14

### CODE REVIEW

File: lib/build.gradle [hunk 1]
```diff
@@ -36,8 +36,9 @@ android {
 }
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
```
Comment (keiththompson) [inline lib/build.gradle:39]: Wasn't importing anything.

File: lib/src/main/java/com/ragnarok/rxcamera/SurfaceCallback.java [hunk 2]
```diff
@@ -1,7 +1,6 @@
 package com.ragnarok.rxcamera;
 
 import android.graphics.SurfaceTexture;
-import android.util.Log;
```
Comment (keiththompson) [inline lib/src/main/java/com/ragnarok/rxcamera/SurfaceCallback.java:4]: Unused import.

**Comment by ragnraok** [comment]
Thanks, but for the camera related projects, it seem that it is a little bit hard to write the testcases, 
like the flash setting, to figure out if work normally, you need to do it manually

**Comment by keiththompson** [comment]
@ragnraok Sure, but I don't believe that should be a blocker to having _any_ unit tests? There should be unit testing around basic classes like this one or other models / utility methods. I agree instrumentation tests would be difficult because of the Camera but I think it's a slightly separate issue to this.

**Comment by ragnraok** [comment]
you are right, I will consider to add some test later

**Comment by keiththompson** [comment]
💭 Shall I close this pull request? I don't see what the blocker is on this.

**Comment by ragnraok** [comment]
I close it

**Pull request closed without merging**